src/registry/billingsdk/plan-change-calculator.tsx

```
@@ -0,0 +1,590 @@
+  const [error, setError] = useState<string | null>(null);
+  const [selectedDateOption, setSelectedDateOption] = useState<'today' | 'tomorrow' | 'nextWeek' | 'nextMonth' | 'custom'>('today');
+  const [customDate, setCustomDate] = useState<string>(new Date().toISOString().split('T')[0]);
+
+  const getEffectiveDate = useMemo(() => {
+    const today = new Date();
+    today.setHours(0, 0, 0, 0);
+    switch (selectedDateOption) {
+      case 'today':
+        return today;
+      case 'tomorrow':
+        const tomorrow = new Date(today);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)